src/components/MediaPlayer/VideoJS/components/js/VideoJSNextButton.js

```diff
@@ -18,6 +18,7 @@ class VideoJSNextButton extends vjsComponent {
   constructor(player, options) {
     super(player, options);
     this.setAttribute('data-testid', 'videojs-next-button');
+    this.addClass('vjs-play-control vjs-control vjs-next-btn');
```

Comment:
What's the difference between `vjs-next-btn` and `vjs-next-button`?  Are they supposed to be the same?  (Same question for `vjs-previous-btn` and `vjs-previous-button`.)

Reply:
I think these classes are from an experimental step in the implementation, which was supposed to be removed. Good catch! Let me remove them and re-commit the code.